test/e2e/velero_utils.go

```diff
@@ -272,16 +277,36 @@ func VeleroInstall(ctx context.Context, veleroImage string, veleroNamespace stri
 			return errors.New("No object store provider specified - must be specified when using kind as the cloud provider") // Gotta have an object store provider
 		}
 	}
+
+	providerPlugins := getProviderPlugins(objectStoreProvider)
+
+	// TODO - handle this better
+	if cloudProvider == "vsphere" {
```

Comment:
It seems to me this should be grouped with the if/else that starts on ln 270. Pls add a line before the call to `err := EnsureClusterExists(ctx)`. That call should actually be the very first thing in this func.